crayonball-ruby.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/meson.build b/meson.build
+index 9b423f3..50e5247 100755
+--- a/meson.build
++++ b/meson.build
+@@ -68,7 +68,7 @@ exe = executable('aerosol',
+ 		'Aerosol/ext/Photon/ruby/ext/rb_photon_chipmunk.c',
+ 	],
+ 	include_directories: [
+-		include_directories(ruby_dir),
++		include_directories('/app/lib/ruby/1.8/x86_64-linux'),
```

Comment:
is that why you excluded aarch64?

Reply:
Indeed it was, it should be fixable with a few modifications to how CrayonBall finds the path, or just hardcoding the path and switching based on arch. It'll also need to fix https://github.com/slembcke/CrayonBall/blob/9bcab497c890e96e12788734bf0fdaa0c2a97050/Aerosol/ext/Chipmunk/CMakeLists.txt#L33

Reply:
Yeah. I have more experience with CMake, but wanted to try Meson for once. A bit out of my element making it do anything other than basic builds, but I can figure that out later. I want to make it control better with touchscreens too.